pkg/test/framework/components/istio/operator.go

```diff
@@ -30,6 +30,16 @@ import (
 	"istio.io/istio/pkg/test/scopes"
 )
 
+var (
+	multiclusterOptions = []string{
+		"--set", "multiCluster=enabled: true",
+		// Installs Istio's CoreDNS to enable *.global resolution.
+		"--set", "addonComponents.istiocoredns.enabled=true",
```

Comment:
is this the prod ready/recommended config for users? should we be moving this to more to core istio?

Reply:
We've documented it as the only way to do replicated control planes (see https://istio.io/docs/setup/install/multicluster/gateways/#setup-dns). The istiocoredns plugin itself hasn't been updated in 12 months (see https://github.com/istio-ecosystem/istio-coredns-plugin).

There are other simpler ways to setup replicated control planes, partiacultly if the clusters are on the same network. For example:

```bash
# install intermediate roots in each cluster
istioctl manifest apply -f --context=<cluster0>
istioctl manifest apply -f --context=<cluster1>
istioctl x create-remote-secret --context=<cluster0> | kubectl apply -f - --context=<cluster1>
istioctl x create-remote-secret --context=<cluster1> | kubectl apply -f - --context=<cluster0>
# fini
```

The only thing different with istio.io/docs/setup/install/multicluster/gateways is the *.global trickery (ServiceEntry+istiocoredns+TCP rewrite filter).